src/main/migrations/changesets/db_schema.xml

```diff
@@ -0,0 +1,158 @@
+                <constraints nullable="false"/>
+            </column>
+            <column name="STATUS" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="JES_JOB">
+            <column autoIncrement="true" name="JES_JOB_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_JES_JOB"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JES_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JES_STATUS" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="LOCAL_JOB">
+            <column autoIncrement="true" name="LOCAL_JOB_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_LOCAL_JOB"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="COMMAND" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="RC" type="INT"/>
+        </createTable>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
+    </changeSet>
+
+    <changeSet id="db_schema_symbol_table_mysql" author="scottfrazer" dbms="mysql">
+        <createTable tableName="SYMBOL">
+            <column autoIncrement="true" name="SYMBOL_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_SYMBOL"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="SCOPE" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="NAME" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ITERATION" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="IO" type="ENUM('INPUT', 'OUTPUT')">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WDL_TYPE" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WDL_VALUE" type="TEXT"/>
+        </createTable>
+
+        <modifySql>
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
+    </changeSet>
+
+    <changeSet id="db_schema_symbol_table_otherdb" author="scottfrazer" dbms="!mysql">
```

Comment:
Is the only difference between these `mysql` and `!mysql` changesets the type of the `IO` column?  If so, you might be able to collapse these into a single changeset with properties.  I have this for the wdl/inputs aux table I'm working on now:

```

    <property name="clob.type" value="LONGTEXT" dbms="mysql"/>
    <property name="clob.type" value="LONGVARCHAR" dbms="hsqldb"/>
...
        <createTable tableName="WORKFLOW_EXECUTION_AUX">
            <column name="WORKFLOW_EXECUTION_ID" type="INT"/>
            <column name="WDL_SOURCE" type="${clob.type}"/>
            <column name="WDL_INPUTS" type="${clob.type}"/>
        </createTable>
```